src/guiapi/include/IWindowMenuItem.hpp

```diff
@@ -28,23 +29,26 @@ class IWindowMenuItem {
 protected:
     virtual void printIcon(IWindowMenu &window_menu, Rect16 rect, uint8_t swap, color_t color_back) const;
     void printLabel_into_rect(Rect16 rolling_rect, color_t color_text, color_t color_back, const font_t *font, padding_ui8_t padding, uint8_t alignment) const;
-    virtual void printText(IWindowMenu &window_menu, Rect16 rect, color_t color_text, color_t color_back, uint8_t swap) const;
+    virtual void printItem(IWindowMenu &window_menu, Rect16 rect, color_t color_text, color_t color_back, uint8_t swap) const;
     virtual void click(IWindowMenu &window_menu) = 0;
     virtual Rect16 getRollingRect(IWindowMenu &window_menu, Rect16 rect) const;
+    virtual std::array<Rect16, 2> getMenuRects(IWindowMenu &window_menu, Rect16 rect) const = 0;
```

Comment:
using std::pair or a tuple is more common in this case ... but never mind, the resulting code will be similar

Reply:
rewritten